vnext/Microsoft.ReactNative/JsiWriter.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "jsi/jsi.h"
+#include "winrt/Microsoft.ReactNative.h"
+
+namespace winrt::Microsoft::ReactNative {
+
+struct JsiWriter : winrt::implements<JsiWriter, IJSValueWriter> {
+  JsiWriter(facebook::jsi::Runtime &runtime) noexcept;
+  facebook::jsi::Value MoveResult() noexcept;
+
+ public: // IJSValueWriter
+  void WriteNull() noexcept;
```

Comment:
>WriteNull [](start = 7, length = 9)

I know that the JsValueWriter also just has null, but should we have 'undefined' as well since null and undefined are separate concepts in JS?

Reply:
Has an API for interop with other languages like C++ and C#, I don't think this is necessary, because these languages only have one kind of null. Having undefined doesn't improve type projection.